mut/redirects/redirect_main.py

```diff
@@ -121,8 +122,14 @@ def parse_source_file(source_path: str, output: str) -> None:
                 if keyword_split[0] == 'symlink':
                     type_split = line.split(':', 1)
                     sym_split = [sym.strip() for sym in type_split[1].split('->')]
-                    sym_tup = (sym_split[0], sym_split[1])
-                    rc.symlinks.append(sym_tup)
+                    alias = sym_split[0]  # did not exist previously
+                    origin = sym_split[1]  # original folder
+
+                    if os.path.islink('./build/public/' + alias):
```

Comment:
In general, it's bad juju to do checks like this before creating a file because of the risk of race conditions (probably not a serious concern in this application, though).

Honestly, what I'd just do here is unconditionally try to remove that path, and just ignore `FileExistsError`: 
```python
alias_path = os.path.join('build/public', alias)
try:
    os.remove(alias_path)
except FileExistsError:
    pass
```